test/widgets/top_navigation_test.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import 'package:time_tracker/src/widgets/top_navigation.dart';
+
+void main() {
+  testWidgets('Top Navigation test', (WidgetTester tester) async {
+    await tester.pumpWidget(MaterialApp(
+        title: 'dummy_app',
+        home: Scaffold(
+          appBar: TopNavigation(),
+          body: Container(),
+        )));
+    expect(find.byKey(Key('task_list_icon')), findsOneWidget);
```

Comment:
It's nice to have comments for front-end tests. I recommend something like:
`// Click task list icon and assert page navigates to Task List Screen`